topics/imaging/tutorials/detection-of-mitoflashes/workflows/Capturing-mitoflashes-tests.yml

```diff
@@ -0,0 +1,9 @@
+- doc: Test outline for Capturing-mitoflashes
+  job: {}
+  outputs:
+    Spot Detection of Mitoflashes:
```

Comment:
For the data output, I think maybe "Spot detection for mitoflash events" would be better. I had to give the output label names because of the test file.